src/vl-upload.js

```diff
@@ -384,4 +402,23 @@ export class VlUpload extends vlFormValidationElement(vlElement(HTMLElement)) {
   _enableAutoProcessQueue() {
     this._dropzone.options.autoProcessQueue = true;
   }
+
+  _addSlots() {
+    Promise.all([this.uploadElement]).then(() => {
+      const subTitleSlotNodes = this._subTitleSlotElement.assignedNodes();
+      if (subTitleSlotNodes.length > 0) {
+        const small = this.uploadElement.querySelector('small');
+        small.innerHTML = subTitleSlotNodes[0].outerHTML;
```

Comment:
lijkt me iets wat je zou encapsuleren in een getter (voor de 'small' te selecteren) + een setter voor zijn html te setten

Reply:
goeie toevoeging bij deze aangepast